src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Figure {
+    void draw();
+}

```

Comment:
According to the checklist, an interface should not contain methods that are conceptually different. If there's another method like `getArea()` to be included in this interface, it should be split into separate interfaces as `draw()` and `getArea()` do not conceptually align.